content/guest-essays/uncertainty-and-utilitarianism.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: "Uncertainty and Utilitarianism"
 book: "Uncertainty and Utilitarianism"
-year: "2022"
-date: 2022-08-08T10:10:55-04:00
+year: "2023"
+date: 2023-01-01T10:10:55-04:00
 author: "Krister Bykvist"
 draft: false
 layout: "book"
 menu: "guest"
 weight: 8
-image: "/img/essays/criminal-law.jpg"
+image: "/img/essays/_______________________.jpg"
```

Comment:
@rychappell we'll want a fitting image here, perhaps https://unsplash.com/photos/PXB7yEM5LVs ?